EESSI-pilot-install-software.sh

```diff
@@ -186,36 +186,42 @@ else
     echo_green ">> MODULEPATH set up: ${MODULEPATH}"
 fi
 
-for eb_version in '4.7.2' '4.8.0' '4.8.1'; do
+# assume there's only one diff file that corresponds to the PR patch file
+pr_diff=$(ls [0-9]*.diff | head -1)
+
+# use PR patch file to determine in which easystack files stuff was added
+for easystack_file in $(cat ${pr_diff} | grep '^+++' | cut -f2 -d' ' | sed 's@^[a-z]/@@g' | grep '^eessi.*yml$'); do
```

Comment:
That's ok. However, if we can assume that `git` is available we could also just do

```bash
for easystack_file in $(git diff --name-only | grep '^eessi.*yml$'); do
```

Reply:
Wouldn't `git diff` also list files that were removed?
With `+++` I'm making sure we only get files where stuff was added (so the file will exist).

May not matter much in practice, as long as we don't remove easystack files (I think we may do that when rebuilding for `eessi.io`, but we could do a cleanup PR first to collapse easystack files to latest EasyBuild release, and then follow up with "build" PRs for `eessi.io`).

So I'll switch to this, it's a lot cleaner (and hopefully it's only temporary, since I would really prefer getting this info from the bot instead).

Reply:
fixed in [4657384](https://github.com/EESSI/software-layer/pull/351/commits/46573849c7030c9ec9c2d6b9baa7cc92ec48b7ab)